lib/dill/checkpoint.rb

```diff
@@ -11,7 +11,7 @@ class << self
       attr_accessor :rescuable_errors
     end
 
-    self.rescuable_errors = [StandardError]
+    self.rescuable_errors = [StandardError, Exception]
```

Comment:
It looks like Dill requires Capybara, which requires RSpec, so Dill effectively requires RSpec. I wonder if we should just add the RSpec exception here ourselves, though it would also be nice to offer other folks the ability to customize if another error subclasses Exception instead.